cmd/osbuild-service-maintenance/aws.go

```diff
@@ -84,33 +126,212 @@ func AWSCleanup(maxConcurrentRequests int, dryRun bool, accessKeyID, accessKey s
 		wg.Wait()
 	}
 
+	// using `errors` to collect all errors as we want to
+	// continue execution if only one cleanup fails
+	var errors MultiError
```

Comment:
Quick drive-by, there is `errors.Join()` now, the more common way to write this (IME) is:
```go
var errs []error
...
errors = append(errs, err)
...
return errors.Join(errs)
```